src/file-server.js

```diff
@@ -12,18 +12,22 @@ const createFileServer = (mountDirectory) => {
     };
 
     const contentType = mimeTypes[extname] || 'application/octet-stream';
+    const headers = {
+      'Content-Type': contentType, 
+      'Access-Control-Allow-Origin': '*'
```

Comment:
Bumping puppeteer means it respects Cors policies so the server needed to be setup to allow Cross Origin requests.

Reply:
I assume there is no problem with having this as `*` instead of e.g. just allowing e.g. only the puppeteer server to access?

Reply:
It is all contained inside a node server on localhost so this shouldn't be a problem.